code/modules/power/apc.dm

```diff
@@ -462,6 +468,16 @@
 
 		if(QUALITY_PRYING)
 			if(opened)
+				if (emp_shield)
+					if (prob(50) && electrocute_mob(usr, terminal.powernet, terminal))
+						return
+					if (!do_after(usr, 20 SECONDS, src)) // More coils = takes longer to disassemble. It's complex so largest one with 5 coils will take 50s
```

Comment:
Why !do_after with a return instead of if(do_after) and what comes afterwards?

Reply:
Also the comment line does not match with what the duration states

Reply:
```
if(!do_after)
    return
do_something
```
is the same as
```
if(do_after)
    do_something
```

Oh, and about comment - this block was copy-pasted from SMES coils insertions, didn't pay attention to comments